### ISSUE TITLE
description and isFavoriteByDefault do not update

### ISSUE THREAD

**Issue by dbaldacchino** [microsoftgraph/microsoft-graph-docs#7378]
Hi, I am trying to update a standard channel. The displayName property updates just fine but both description and isFavoriteByDefault do not update. I am using the beta endpoint. Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ef42e99a-4f43-cd17-e512-24040e2fd81c
* Version Independent ID: bc912754-3388-4cc0-90cc-df7164a78fc1
* Content: [Patch channel - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/channel-patch?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/channel-patch.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/channel-patch.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @clearab
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Oweeboi011** [comment]
Hello,
We are trying to use the 'PATCH Channel API' as well and we keep on having an "ERROR 400" message. Any leads on how to fix this?  We checked our request header and body and they are all correct based on the documentation given. 

Thanks for your quick reply.

**Comment by ashoksubedi1** [comment]
Same here
I am updating isFavoriteByDefault to "true" to existing channels and I do get request i still get isFavoriteByDafault comes "null". I thought I had to create a channel and set it to true while creation of the channel but even when I set it to true while creating a new channel, still the same gives me the "null" value. This is a bug. By the way is there a way to make group members follow the channel by default just like Favorite but with the follow.
